ostree/ostree_selinux.go

```diff
@@ -0,0 +1,78 @@
+// +build !containers_image_ostree_stub,!mac_stub
```

Comment:
Please name the tag `containers_image_mac_stub`, this is a library and we don’t want to stomp on calling applications’ tags.